aspnetcore/migration/20_21.md

```diff
@@ -55,7 +55,7 @@ The version specified by the package reference is the *minimum required* version
 
 Known issues for projects targeting the shared runtime:
 
-* It's difficult to use the 2.1.0 `Microsoft.AspNetCore.App` package with Entity Framework Core 2.1.1. The recommended solution is to upgrade `Microsoft.AspNetCore.App` to 2.1.1. For more information, see [Packages that share dependencies with Microsoft.AspNetCore.App cannot reference patch versions](https://github.com/aspnet/Universe/issues/1180).
+* Using `Microsoft.AspNetCore.App` 2.1.0 with Entity Framework Core 2.1.1 causes NuGet restore failures due to package conflicts.. The recommended solution is to upgrade `Microsoft.AspNetCore.App` to 2.1.1. For more information, see [Packages that share dependencies with Microsoft.AspNetCore.App cannot reference patch versions](https://github.com/aspnet/Universe/issues/1180).
```

Comment:
Two periods.

Reply:
dang it

Reply:
DOE! Bart! lol ....... I do crap like that all the time. If @scottaddie had a nickel for every one of my *GooberRex* foul-ups, he'd be a 💰 **rich man indeed!** 💰